demo/site/src/documents/pages/blocks/ContentGroupBlock.tsx

```diff
@@ -38,7 +39,7 @@ const ContentGroupContentBlock = withPreview(
 
 export const ContentGroupBlock = withPreview(
     ({ data: { content, backgroundColor } }: PropsWithData<ContentGroupBlockData>) => (
-        <PageLayout className={styles[backgroundColor]}>
+        <PageLayout className={clsx(backgroundColor === "darkGray" && styles.darkGray, backgroundColor === "lightGray" && styles.lightGray)}>
```

Comment:
backgroundColor can also be "default". We could also use `className={backgroundColor !== "default" ? styles[backgroundColor] : undefined}`. What do you prefer?

Reply:
both are ok. Used you variant now, bit more elegant ;)